tests/integration/test_postprocessing.py

```diff
@@ -15,39 +15,39 @@
 
 def test_postprocessing():
     """Integration test for bids-postprocessing executable"""
-    xnat_user = os.environ.get("XNAT_USER", "testuser")
+    xnat_user = os.environ.get("XNAT_USER", "")
     xnat_pass = os.environ.get("XNAT_PASS", "")
-    session = "XNAT2_E00017"
+    session = "XNAT_E00114"
     bids_root_dir = os.environ.get("XNAT_BIDS_ROOT", "./tests/xnat2bids")
-    seqlist = ["19", "23", "24"]
 
     if os.path.exists(bids_root_dir):
         shutil.rmtree(bids_root_dir, ignore_errors=True)
 
     os.mkdir(bids_root_dir)
 
-    xnat2bids_cmd = f"{session} {bids_root_dir} -u {xnat_user} -p {xnat_pass} \
-                      -i {' -i '.join(seqlist)}"
+    xnat2bids_cmd = f"{session} {bids_root_dir} -u {xnat_user} -p {xnat_pass}"
 
     xnat2bids_split_cmd = shlex.split(xnat2bids_cmd)
 
     r = runner.invoke(xnat2bids_app, xnat2bids_split_cmd)
     print(r.stdout)
 
-    bids_dir = "tests/xnat2bids/ashenhav/study-1222/bids"
+    bids_dir = "tests/xnat2bids/bnc/study-demodat/bids"
 
     bids_postprocess(
         session,
         bids_dir,
         log_file="",
-        session_suffix="01",
-        includesubj=[9011, 9999, 111, 9999, 111],
-        skipsubj=[9999, 111],
+        session_suffix="SESSION1",
+        includesubj=[],
+        skipsubj=[],
         verbose=0,
     )
 
     # lazy check for the intendedFor field in one json file
-    json_file = f"{bids_dir}/sub-9011/ses-01/fmap/sub-9011_ses-01_acq-greAP_phasediff.json"
+    json_file = (
```

Comment:
Maybe you can make an issue about if, but we can definitely improve this test to check more files. Maybe as part of the more through testing of the behavior of the intended for